BLECarPiZeroW/dbusObjects/GPSSensorDBUS.py

```diff
@@ -32,7 +32,10 @@ def _startSensor(self):
     def getCoords(self):
         """returns the GPS object from the sensor"""
         gps = self.gpsSensor.get_gps()
-        retVal={'latitude':gps.latitude, 'longitude':gps.longitude, 'altitude':gps.altitude}
+        if gps.latitude:
+            retVal={'latitude':gps.latitude, 'longitude':gps.longitude, 'altitude':gps.altitude,'real':1.0}
+        else:
```

Comment:
I suppose this is for debugging only

Reply:
If the default value is returned dBus complains about the type returned, blocking the endpoint.
Something more elegant could be done, ofc